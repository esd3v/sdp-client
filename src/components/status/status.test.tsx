import {Status} from './';
import renderer from 'react-test-renderer';
import React from 'react';

test('renders correctly', () => {
  const tree = renderer
    .create(<Status type="normal" message="test"/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
