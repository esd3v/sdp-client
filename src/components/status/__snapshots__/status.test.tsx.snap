// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.emotion-0 {
  padding: 16px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  color: false;
  box-shadow: 0 1px 3px rgba(0,0,0,0.12),0 1px 2px rgba(0,0,0,0.24);
  background-color: #fff;
  margin-bottom: 16px;
}

<div
  className="emotion-0"
>
  test
</div>
`;
